cmd/release/docs/new_docs/input.go

```diff
@@ -0,0 +1,59 @@
+package new_docs
+
+import (
+	"bytes"
+	"github.com/aws/eks-distro/cmd/release/utils"
+)
+
+type NewDocInput struct {
+	FileName       string
+	TemplateWriter bytes.Buffer
+	AppendToEnd    func() (string, error)
+}
+
+type releaseInfo interface {
+	Tag() string
+	ManifestURL() string
+}
+
+func CreateNewDocsInfo(ri releaseInfo) ([]NewDocInput, error) {
+	changeLogWriter, err := getTemplateWriter(ri, changelogTemplateInput)
+	if err != nil {
+		return []NewDocInput{}, err
```

Comment:
consider adding fmt'd errors to these with a bit of info (e.g. 'failed to create changeLogWriter template writer'), so we know call to `getTemplateWriter` failed